### PR TITLE
Add B902: blind except Exception: statement

### LINKED ISSUE #3: except: in comments
It would be great if flake8-blind-except could handle the case that a inline or multiline comment has a `except:` in it.

As _except_ is a regular English word is not so uncommon to have it on comments and maybe (happened to me) followed by a colon.

### LINKED ISSUE #4: except Exception should be also avoided
almost all Python exceptions [inherit from Exception](https://docs.python.org/2/library/exceptions.html#exception-hierarchy).

so, from QA point of view, using `except Exception:` is almost as bad as using `except:`.

### LINKED ISSUE #11: Consider deprecating in favour of built-in E722
Hi,

Thanks for creating this plugin; it's very useful. However it appears to have been superseded by [a change in `pycodestyle`](https://github.com/PyCQA/pycodestyle/commit/543f12b06592c53e2e60edc4846ee02ab9550e8b) and so equivalent functionality is now available in `flake8` without a plugin.

Would you be open to a PR which updates the README to point at that error instead?

### PR DESCRIPTION
**Pull request by ddelange**
Fixes #3, fixes #4, closes #11.

Added tests for old and new cases, updated README, and bumped `__version__`.

Here is my regex playground: https://regex101.com/r/mGeuK1/5

@elijahandrews is there anything else I can do? Do you have time to push a new release to pypi? I can also offer to become github and/or pypi maintainer, on your invitation of course.

### CODE REVIEW

**Comment by ddelange** [comment]
I just saw #11.

With that knowledge, the built-in might be worth noting on the top of the README, adding that the added functionality from this PR is *not* built-in, making the install worthwhile for zealous users :)

Continuing on that thought, I will also add `BaseException` to the mix.

File: flake8_blind_except.py [hunk 1]
```diff
@@ -1,21 +1,41 @@
...
-BLIND_EXCEPT_REGEX = re.compile(r'(except:)')  # noqa
+__version__ = '0.2.0'
 
+BLIND_EXCEPT_REGEX = re.compile(r'(^[ \t]*except(.*\b(Base)?Exception\b.*)?:)')  # noqa
 
 def check_blind_except(physical_line):
+    """Check for blind except statements.
+
+    >>> check_blind_except('except:')
+    (0, 'B901 blind except: statement')
```
Comment (elijahandrews) [inline flake8_blind_except.py:15]: I like these test cases! Would you be interested in putting them in `flake8_blind_except_test.py` file that we can drive with [unittest](https://docs.python.org/3/library/unittest.html) (or some other python testing lib if you prefer?)

I can do it later if you're not interested.

**Review by elijahandrews — COMMENTED** [review]
Hey @ddelange, thanks for the contribution. I like this change. Just one comment about whether you'd be interested in adding a way to automatically run the test cases you added in the docstring.

File: flake8_blind_except.py (hunk 1 above)
Comment (elijahandrews) [reply flake8_blind_except.py:15]: Oh sorry, I missed that we can actually run these tests with the pytest command you provided, never mind!

**Review by elijahandrews — APPROVED** [review]
Thanks!

File: flake8_blind_except.py (hunk 1 above)
Comment (ddelange) [reply flake8_blind_except.py:15]: The good ol' [`doctest`](https://docs.python.org/2.7/library/doctest.html) module still going strong :)

Auto tests are actually not too much work either, and on public repos you can initiate branch protection using it. If you're interested, you could add a basic CI.yml file like in e.g. https://github.com/cpcloud/ipython-autotime/pull/17

**Comment by elijahandrews** [comment]
@ddelange I believe I've pushed v0.2.0 to pypi. It's been years since I've done this; may you please double check that I've done it correctly?

**Comment by ddelange** [comment]
Confirming all green, sdist has been uploaded 💥  https://pypi.org/project/flake8-blind-except/#files

**Pull request merged**